release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1294,10 +1294,14 @@ Hosted control planes for {product-title} is now Generally Available on the bare
 ==== Creating ARM NodePool objects on AWS hosted clusters (Technology Preview)
 In this release, you can schedule application workloads on 64-bit ARM and AMD64 from the same hosted control plane. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-cluster-arm-node-pools[Creating ARM NodePool objects on AWS hosted clusters].
 
-[id="ocp-4-14-hcp-ibm-power-z"]
+[id="ocp-4-14-hcp-ibm-z"]
 ==== Hosted control planes on {ibm-z-title} (Technology Preview)
 In this release, hosted control planes is available as a Technology Preview feature on {ibm-z-title}. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosting cluster on 64-bit x84 bare metal for IBM Z compute nodes (Technology Preview)].
 
+[id="ocp-4-14-hcp-ibm-power"]
```

Comment:
@alishaIBM @SNiemann15 The content in lines 1297 - 1299 is currently under review in PR #68676. You can remove those lines from this PR. Thank you!

Reply:
@lahinson okay, I have updated the PR. cc: @SNiemann15

Reply:
Thank you, @alishaIBM!